bug-fix commits

incpi opened the pull request:
Hello @dbeck121 once again,

Cleaned-up code for #103 - closed because my issue with git.   Do let me know if you found any issue. 
apart from Clipping Text - unrestricted width in Last visited section. (Fix - maybe in future.).

for this PR,

- Again, sidebar UI changes. (bit cleaner imo ). 
- Some more controls
- #101 issue fixed.

Done with issues. You can merge it if you want. @fippu82 Thank you for your help. let me know if you found any other bug.
I don't have anything else to add it for now. Thank you for being patient with me.

Thanks,
Omkar

fippu82 commented:
@incpi, thank you! I have one more small design issue: 
It's not intuitively clear what the status of this toggle button is: 
![image](https://github.com/dbeck121/CPI-Helper-Chrome-Extension/assets/14195212/01ae1029-98d2-4b16-a7e8-b9f8f3e5f32b)

It should clearly reflect the current status. E.g. like this:
https://www.freepik.com/icon/toggle-button_5720465

Do you mind to change this?

incpi commented:
Thank you for your feedback. I have revised the text to read "Set as <Action>". You may wonder why I did not use icons instead. The reason is that icons do not fit well with the UI default component. I tried using icons at first, but they looked unappealing to me. So, I decided to use text instead. 

I hope now, this clears up confusion of UI.

fippu82 commented:
@incpi  That does the trick as well, thank you! 
One more thing: I notice that text links on CPI pages are now colored green, like this: 

![image](https://github.com/dbeck121/CPI-Helper-Chrome-Extension/assets/14195212/f8f10684-e8e2-4b3d-9b54-520cfe9e51a4)

Was that intended?

incpi commented:
@fippu82 ,

Sorry, that was my mistake. I was testing a new design. You can remove popup.css  from the manifest file or you can try along with this current pr. I fixed a lot of issues and added new features in this Pr.

Hope this helps.
BR,
Omkar

dbeck121 commented:
Hi, thanks for the development! I will test and it will be probably part of next release! :-)

dbeck121 commented:
@incpi I checked it and works great! It will be part of next release. It makes CPI Helper a little bit better and a lot nicer. Thank you so much for your effort! The CPI Community will thank you!
